auto-generated/cluster/stone-prd-rh01/tenants/oeo-cicada-tenant/appstudio.redhat.com_v1alpha1_releaseplan_oeo-release-as-service.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlan
+metadata:
+  labels:
+    release.appstudio.openshift.io/auto-release: "true"
+    release.appstudio.openshift.io/standing-attribution: "true"
+  name: oeo-release-as-service
+  namespace: oeo-cicada-tenant
+spec:
+  application: osd-example-operator
```

Comment:
I believe this should be for your application, not the example one, no?

Reply:
Hey @johnbieren, I'm part of the CICD team for service delivery and we have an operator named "osd-example-operator" that we use as a test bed for new things. I'm trying to understand the process with OEO first before moving on. See openshift/osd-example-operator

Reply:
Ah, got it